src/core_types.jl

```diff
@@ -28,6 +28,7 @@ rotation_angle(r::Rotation{3}) = rotation_angle(AngleAxis(r))
 rotation_axis(r::Rotation{3}) = rotation_axis(AngleAxis(r))
 
 # `convert` goes through the constructors, similar to e.g. `Number`
+Base.convert(::Type{R}, rot::R) where {N,R<:Rotation{N}} = rot
```

Comment:
I had assumed this was covered by `Base.convert(::Type{T}, x::T) where {T} = x`? The specialization rules confuse me.